src/classes/UTIL_MasterSchedulableHelper.cls

```diff
@@ -124,7 +124,11 @@ public without sharing class UTIL_MasterSchedulableHelper {
     public static void setScheduledJobs() {
 
         Map<String, CronTrigger> scheduledJobs = new Map<String, CronTrigger>();
-        
+
+        // always delete and reschedule all jobs in this particular scenario
+        Error_Settings__c orgErrorSettings = UTIL_CustomSettingsFacade.getOrgErrorSettings();
+        Boolean rescheduleAllJobs = (UTIL_Namespace.isSandbox() && orgErrorSettings.Sandbox_Jobs_Rescheduled__c == false);
```

Comment:
Ah. Thanks for clarifying. I suppose those could be combined into another boolean, but `rescheduleAllJobs` is referenced elsewhere by itself and the larger condition (`rescheduleAllJobs || currentJob.CreatedBy.Name == 'xx' || !currentJob.CreatedBy.IsActive)`) is only used that one time in the loop (where the values change each time).